db/primary_migrate/20221118191134_add_confirmed_to_identities_and_agency_identities.rb

```diff
@@ -0,0 +1,5 @@
+class AddConsentedToAgencyIdentities < ActiveRecord::Migration[7.0]
+  def change
+    add_column :agency_identities, :consented, :boolean
```

Comment:
had a chat with @mitchellhenke and we realized that we probably don't need this. Agency UUIDs don't go away when a user unlinks all the apps, so we don't need to "hide" AgencyIdentity rows like this, which will help us simplify